app/views/comments/_form.html.erb

```diff
@@ -0,0 +1,12 @@
+<% @comment[:blog_id] = @blog %>
+<% @comment[:user_id] = @user %>
+
+
+<%= form_for [@blog, @comment] do |f| %>
+
+ <p> <%= f.label :content %><br />
```

Comment:
HTML5 gives us a nifty `<fieldset>` tag for these -- nice specificity without having to mess with extra selectors.